src/components/search/search.tsx

```diff
@@ -87,12 +87,9 @@ export default function Search({
                   </div>
                   <span className={style.itemName}>
                     {highlightSearchText(inputValue, item.name).map(
```

Comment:
I think all this can be simplified to just put the `item.name` now, no need for any of the highlighting stuff since we are to remove it.